pages/faq.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: FAQ
```

Comment:
Should the title be `Frequently Asked Questions` or `FAQ`? This is what appears in the tab name in the browser.